portfolio/src/main/webapp/script.js

```diff
@@ -226,3 +227,11 @@ async function deleteComments() {
   await fetch("/delete-data", { method: "POST" });
   displayServletContent(0);
 }
+
+/* Creates a map and adds it to the page */
+function initializeMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 42.453, lng: -76.474}, zoom: 16}
```

Comment:
Where is this latlng? Can you put it in a constant and document what is this place?

Reply:
Since this PR is already merged in, I will add this in the next diff.